dev/ProgressRing/ProgressRing.xaml

```diff
@@ -2,68 +2,99 @@
-                                <PathGeometry>
-                                    <PathGeometry.Figures>
-                                        <PathFigureCollection>
-                                            <PathFigure x:Name="OutlineFigurePart">
-                                                <PathFigure.Segments>
-                                                    <PathSegmentCollection>
-                                                        <ArcSegment
-                                                            x:Name="OutlineArcPart"
-                                                            IsLargeArc="True"
-                                                            SweepDirection="Clockwise"/>
-                                                    </PathSegmentCollection>
-                                                </PathFigure.Segments>
-                                            </PathFigure>
-                                        </PathFigureCollection>
-                                    </PathGeometry.Figures>
-                                </PathGeometry>
-                            </Path.Data>
-                        </Path>
-                        <!-- Progress Ring -->
-                        <Path
-                            Fill="Transparent"
-                            Stroke="{TemplateBinding Foreground}"
-                            StrokeThickness="{TemplateBinding StrokeThickness}"
-                            StrokeStartLineCap="Round"
-                            StrokeEndLineCap="Round">
-                            <Path.Data>
-                                <PathGeometry>
-                                    <PathGeometry.Figures>
-                                        <PathFigureCollection>
-                                            <PathFigure x:Name="RingFigurePart">
-                                                <PathFigure.Segments>
-                                                    <PathSegmentCollection>
-                                                        <ArcSegment
-                                                            x:Name="RingArcPart"
-                                                            IsLargeArc="False"
-                                                            SweepDirection="Clockwise" />
-                                                    </PathSegmentCollection>
-                                                </PathFigure.Segments>
-                                            </PathFigure>
-                                        </PathFigureCollection>
-                                    </PathGeometry.Figures>
-                                </PathGeometry>
-                            </Path.Data>
-                        </Path>
+
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="CommonStates">
+                                <VisualState x:Name="Normal" />
```

Comment:
>Normal [](start = 53, length = 6)

Why do we have a normal and a determinate state?

Reply:
I believe I missed it when I copied it over; removed.